fecgov/fec-cms
# 2024 Admin fines

## Summary

- Resolves #6043 

Adding the 2024 dates to the admin fines calculator

### Required reviewers

1-2 for code check?

## Impacted areas of the application

The admin fines calculator should no longer offer fines before 2021-12-28.

We've shortened the text in the breadcrumb for the entries that have start & end dates.


## Screenshots

<img width="572" alt="image" src="https://github.com/fecgov/fec-cms/assets/26720877/a47e40c4-5fb5-47ce-a5d5-0907d2db686e">

## Related PRs

None


## How to test

- pull the branch
- `npm run build`
- `./manage.py runserver`
- Go to the [admin fines page](http://127.0.0.1:8000/legal-resources/enforcement/administrative-fines/calculating-administrative-fines/#calculating-a-fine)
- Click the 'Calculate estimated fines' button
- Click 'Start calculating'
- Check that 2021-12-28 is the first available date or
- Check that 2024-01-05 is the most recent date
- Check that selections are working
(internal teams have already confirmed that calculations are correct)
